frontend/packages/console-dynamic-plugin-sdk/src/types.ts

```diff
@@ -1,4 +1,46 @@
-import { Extension } from '@console/plugin-sdk/src/typings/base';
+/**
+ * Console feature flags used to gate extension instances.
+ */
+export type ExtensionFlags = Partial<{
```

Comment:
I copied these types from `@console/plugin-sdk` to get rid of the dependency

Reply:
+1 for reconciling as soon as we can.